velox/exec/fuzzer/ReferenceQueryRunner.h

```diff
@@ -35,6 +35,23 @@ class ReferenceQueryRunner {
       const std::string& sql,
       const std::vector<RowVectorPtr>& input,
       const RowTypePtr& resultType) = 0;
+
+  /// This function will return true if the underlying Query Runner supports
+  /// returning the results of a query in Velox Vector format. If supported
+  /// then 'executeVector' can be called to get results as Velox Vector.
+  bool supportsVeloxVectors() const {
+    return false;
```

Comment:
Curiously why always return false here?

Reply:
At the moment only PrestoQueryRunner supports velox vectors - so I have set the default to false - Anyways I think the main concern is why it isnt virtual and I have added that. Thanks !